pydoctor/model.py

```diff
@@ -752,18 +749,21 @@ def readd(o):
         self.allobjects[fullName] = obj
 
 
-    def getProcessedModule(self, modname):
+    def getProcessedModule(self, modname: str) -> Optional[_ModuleT]:
         mod = self.allobjects.get(modname)
         if mod is None:
             return None
         if isinstance(mod, Package):
-            return self.getProcessedModule(modname + '.__init__').parent
+            initModule = self.getProcessedModule(modname + '.__init__')
+            assert initModule is not None
```

Comment:
Is this "out of test code" assertion required ?

I was hoping that we now have enough unit tests to see them fail if this is None

Reply:
I think the assert was added at some point to tell mypy that `initModule` was safe to call `Module` methods on, but in the method's final form there is nothing called on it, so I'll remove it.